script/lib/create-windows-installer.js

```diff
@@ -22,11 +22,14 @@ module.exports = function (packagedAppPath, codeSign) {
     setupIcon: path.join(CONFIG.repositoryRootPath, 'resources', 'app-icons', CONFIG.channel, 'atom.ico')
   }
 
-  const certPath = path.join(os.tmpdir(), 'win.p12')
-  const signing = codeSign && process.env.ATOM_WIN_CODE_SIGNING_CERT_DOWNLOAD_URL
+  const signing = codeSign && (process.env.ATOM_WIN_CODE_SIGNING_CERT_DOWNLOAD_URL || process.env.ATOM_WIN_CODE_SIGNING_CERT_PATH)
+  let certPath = ATOM_WIN_CODE_SIGNING_CERT_PATH;
+  if (!certPath) {
```

Comment:
Should probably be `if (!certPath && codeSign)` otherwise it's going to download the key files on all builds including ones it wasn't going to sign.